src/Support/HigherOrderMessageCollection.php

```diff
@@ -30,7 +30,7 @@ public function add(string $filename, int $line, string $methodName, array $argu
     public function chain(object $target): void
     {
         foreach ($this->messages as $message) {
-            $target = $message->call($target);
+            $target = $message->call($target) ?? $target;
```

Comment:
Can you add some tests? 

Also, `??` is equal `=== null` correct?

Reply:
Yeah, it will fallback to `$target` if the result of the left-hand value is null or is not set. 👍🏻